nengo_gui/namefinder.py

```diff
@@ -23,8 +23,9 @@ def find_names(self, net):
                    nengo.Connection)
 
         for inst_attr in dir(net):
-            if (not inst_attr.startswith('_')
-                and inst_attr not in base_lists + all_lists):
+            private = inst_attr.startswith('_')
+            in_lists = inst_attr in base_lists + all_lists
+            if (not private and not in_lists):
```

Comment:
I thought because of the `and` the parenthesis were allowed. My bad. When should I use them?